app/assets/constants/places.rb

```diff
@@ -0,0 +1,50 @@
+module Places
+  # some sample destinations (and starting points)
+  DESTINATIONS = {
```

Comment:
A test should probably ensure that these are all `valid_coordinates?` :)

Reply:
I think it is not necessary to test these yet. Sooner or later the coordinates will come from our database which will use integrity checks to ensure that all coordinates are valid for the purpose of our project.